dashboard/src/components/ServiceClassView/ProvisionButton.tsx

```diff
@@ -89,7 +78,7 @@ class ProvisionButton extends React.Component<IProvisionButtonProps, IProvisionB
           isOpen={this.state.modalIsOpen}
           onRequestClose={this.closeModal}
           contentLabel="Modal"
-          style={this.state.displayNameForm ? smallModalStyle : {}}
+          className={this.state.displayNameForm ? "centered-modal" : ""}
```

Comment:
Was the smallModalStyle different from centered? Or fine as is?

Reply:
it was the same but following the inline style syntax